u8timeseries/utils/timeseries_generation.py

```diff
@@ -215,11 +227,11 @@ def us_holiday_timeseries(length: int = 10,
     Returns
     -------
     TimeSeries
-        Binary TimeSeries for US holidays.
+        Binary TimeSeries for country's holidays.
     """
 
     times = pd.date_range(periods=length, freq='D', start=start_ts)
```

Comment:
Could we find a way to have also finer granularities than `D` ? E.g. hourly and minutes?

Reply:
That can come in a future PR though, I guess